### ISSUE TITLE
Training/validation samples for each fold

### ISSUE THREAD

**Issue by ChoiDM** [MIC-DKFZ/nnUNet#1162]
Thank you for the amazing work.

I wonder how to check the specific data composition (training/validation) of each fold from the pre-trained models.

For example, according to [README](https://github.com/MIC-DKFZ/nnUNet#3d-u-net-cascade), 'la_007' belongs to validation set in fold 0 of Task002_Heart.
I don't have any idea to check which samples belong to validation set for the other folds and another task.

Best,
Dongmin Choi.

**Comment by FabianIsensee** [comment]
Hey, just open the splits_final.pkl file in your preprocessed data folder. It is created when you start the first training. 
Also read https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/common_questions.md#creating-and-managing-data-splits

**Comment by ChoiDM** [comment]
Thank you for the reply.

However, I cannot find splits_final.pkl from the pre-trained models, which are downloaded by following https://github.com/MIC-DKFZ/nnUNet#how-to-run-inference-with-pretrained-models.

I want to use the pre-trained weights maintaining the training/validation split not training from scratch.

**Comment by FabianIsensee** [comment]
[splits_final.zip](https://github.com/MIC-DKFZ/nnUNet/files/9548504/splits_final.zip)

there you go :-)
